Bot.py

```diff
@@ -58,22 +63,46 @@ def __init__(self) -> None:
                 log.error("Other API error: {}".format(e))
             exit(1)
 
-        self.interval = Set.Interval(90, self._check)
+        self.interval = Set.Interval(30, self._check)
 
     def _kill(self) -> None:
         """This kills the c̶r̶a̶b̶  bot."""
         self.interval.cancel()
         exit(0)
 
+    def _getUniqueCalls(self, calls) -> list:
+        """Filters the return from the scraper to only tweet unique calls.
+
+        Works by checking if the cache already has that call ID.
+
+        Args:
+            calls (list): The complete list of calls scraped.
+
+        Returns:
+            list: A filtered list of calls.
+        """
+        res = []
+        for call in calls:
+            # If the call is already in the cache skip.
+            if call["id"] in self.cache.keys():

```

Comment:
I _think_ this can just be:
```suggestion
            if call["id"] in self.cache:

```